algrtm/src/components/ContactDescription.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import CustomLink from "./shared/CustomLink";
+
+const ContactDescription = () => {
+  return (
+    <section className="flex flex-col gap-6">
+      <h2 className=" h2 text-neutralLight-neutral10 leading-[4rem] ">
+        Contact
+      </h2>
+      <div className="flex flex-col gap-4">
+        <h6 className="h6 text-neutralLight-neutral20 leading-[1.98rem]">
+          Let's explore how we can work <br /> together. Share your algorithmic
+          <br />
+          ideas, and we’ll provide the
+          <br /> expertise to bring them to life.
+        </h6>
+        <div className="flex gap-4 items-start">
+          {/* TODO add link href */}
+          <CustomLink variant="secondary" text="Book a meeting" href="" />
+
+          {/* TODO add link href */}
+          <CustomLink
+            variant="tertiary"
+            text="Go to the questionnaire"
+            href=""
+            icon="/ArrowUpRightBlack-icon.svg"
+            alt="arrow up icon"
+          />
+        </div>
+      </div>
+    </section>
```

Comment:
There is a problem here with the correct display of the text under "Contact" and the text inside the links:

![image](https://github.com/Nospoko/algrtm/assets/26410548/18e99462-4bd4-48e6-9f45-f980a5dff50c)

First thing - you can add `max-w-[22.375rem] w-full` to `<section className="flex flex-col gap-6">` to set the width to `358px` as in Figma. With this, all `<br />` elements can be removed as the text should fit now

The second thing - `<CustomLink/>` component renders span for an icon even if there is no icon, so this can be fixed with conditional rendering:
```jsx
{icon && (
  <span className='ml-[0.01rem]'>
    {icon && <Icon src={icon} alt={alt} />}
  </span>
)}
```